criu/cr-dedup.c

```diff
@@ -14,7 +14,7 @@ int cr_dedup(void)
 {
 	int close_ret, ret = 0;
 	unsigned long img_id;
-	DIR * dirp;
```

Comment:
pls move code changes to a separate patch.

Reply:
Right, it should be three commits IMHO:
* add scripts and `Makefile` target for coding style updates
* run 'make indent' and commit the change
* add CI/CD checks to verify the coding style

The `indent` target in the `Makefile` could be something like
```
indent:
	find . -name '*.[chS]' -type f -exec clang-format -i {} \;
```

BTW, this would change ~10% of the code: 

```
:~/git/criu$ wc -l $(find . -name '*.[chS]' -type f ) | grep total
 158904 total
:~/git/criu$ git diff --stat | grep "files changed"
 825 files changed, 15641 insertions(+), 18587 deletions(-)

Reply:
Made three commits as suggested.